.ci/install_footloose.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o nounset
+set -o pipefail
+
+cidir=$(dirname "$0")
+arch=$("${cidir}"/kata-arch.sh -d)
+source "${cidir}/lib.sh"
+
+if [ "$arch" != "x86_64" ]; then
+	die "Footloose only available with x86_64."
```

Comment:
`die` feels a bit harsh here maybe? In my head that means we'd bork out on any non-x86 arch and fail completely. Maybe just a `warn` and exit is good enough?